### PR TITLE
Document limitation on CSR auto renewal

### PR DESCRIPTION
**Pull request by pipern**
Document upstream bug https://github.com/krustlet/krustlet/issues/553 and outline how to create a new certificate.

Note I'm not precious about this wording, style, etc. Maybe it doesn't fit in to your current intended style of wording, or maybe the markup doesn't fit the style; no problem on feedback on those, or even to decide to decline the PR.

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)

### CODE REVIEW

File: docs/modules/ROOT/pages/limitations.adoc [hunk 1]
```diff
@@ -5,3 +5,42 @@ The Stackable Agent has limitations, some of which are documented here.
...
+
+```
+Sep 09 13:51:59 server1 stackable-agent[15457]: Caused by:
+Sep 09 13:51:59 server1 stackable-agent[15457]:     certificatesigningrequests.certificates.k8s.io "server1-tls" already exists: AlreadyExists
+```
+
+Remove the current certificate file and restart the agent:
+
+```
+sudo rm /etc/stackable/stackable-agent/secret/agent.crt && sudo systemctl restart stackable-agent
```
Comment (soenkeliebau) [inline docs/modules/ROOT/pages/limitations.adoc:32]: nit: I'd prefer to spread this over two lines, just to be very clear that this is two distinct actions.

**Review by soenkeliebau — DISMISSED** [review]
LGTM - tested the steps and that all worked for me.
Left one really minor comment, but I'd be happy for this to go in as is as well.

File: docs/modules/ROOT/pages/limitations.adoc [hunk 2]
```diff
@@ -5,3 +5,42 @@ The Stackable Agent has limitations, some of which are documented here.
...
+```
+NAME          AGE   SIGNERNAME                      REQUESTOR    CONDITION
+server1-tls   4s    kubernetes.io/kubelet-serving   kube:admin   Pending
+```
+
+This can be approved with `kubectl certificate approve server1-tls` or you can approve all Pending with:
+
+```
+kubectl get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | xargs --no-run-if-empty kubectl certificate approve
+```
```
Comment (siegfriedweber) [inline docs/modules/ROOT/pages/limitations.adoc:46]: Approving everything can be a valid approach on test environments but not on production. This documentation is also meant for production usage so we should either add a warning to this paragraph or remove it.

File: docs/modules/ROOT/pages/limitations.adoc (hunk 1 above)
Comment (pipern) [reply docs/modules/ROOT/pages/limitations.adoc:32]: I think we want to only do the restart if we did successfully remove the file, so I'll put over two lines but keep the &&. If you still prefer two distinct lines, I'll do that instead.

File: docs/modules/ROOT/pages/limitations.adoc (hunk 2 above)
Comment (pipern) [reply docs/modules/ROOT/pages/limitations.adoc:46]: Thanks, I'll tweak.

**Review by siegfriedweber — APPROVED** [review]
LGTM

I processed this PR with antora and the result looks good.

File: docs/modules/ROOT/pages/limitations.adoc (hunk 1 above)
Comment (soenkeliebau) [reply docs/modules/ROOT/pages/limitations.adoc]: That is a fair point, didn't think of that.

**Review by soenkeliebau — APPROVED** [review]
LGTM

**Pull request merged**